### PR TITLE
Fix for CoreRT issue https://github.com/dotnet/corert/issues/3672 (ar…

### PR DESCRIPTION
**Pull request by sergign60**
…mel tizen)

### CODE REVIEW

**Comment by sergign60** [comment]
@Dmitri-Botcharnikov @BredPet @alpencolt @jkotas 
please review

**Comment by sergign60** [comment]
https://github.com/dotnet/coreclr/pull/12104

**Comment by sergign60** [comment]
https://github.com/dotnet/corert/issues/3672

**Comment by sergign60** [comment]
@dotnet-bot test OSX10.12 x64 Checked Build and Test please

File: src/jit/codegenarmarch.cpp [hunk 1]
```diff
@@ -2103,16 +2103,25 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
...
         if (callType == CT_HELPER)
         {
-            // Direct call to a helper method.
-            CorInfoHelpFunc helperNum = compiler->eeGetHelperNum(methHnd);
-            noway_assert(helperNum != CORINFO_HELP_UNDEF);
+// Direct call to a helper method.
+#ifdef FEATURE_READYTORUN_COMPILER
+            if (call->gtEntryPoint.addr != NULL)
+            {
+                addr = call->gtEntryPoint.addr;
```
Comment (jkotas) [inline src/jit/codegenarmarch.cpp]: Does this need to have the case of `accessType == IAT_PVALUE`? 

If not, it would be useful to add assert it at least.

Comment (sergign60) [reply src/jit/codegenarmarch.cpp]: @jkotas I don't see any using of accessType now. May be later when this code will be developed more deeply

**Comment by sergign60** [comment]
@dotnet-bot test Ubuntu16.04 arm Cross Debug Build please

**Comment by sergign60** [comment]
@dotnet-bot test OSX10.12 x64 Checked Build and Test please

**Comment by jkotas** [comment]
cc @dotnet/jit-contrib

**Pull request merged**